src/pages/qna/qna.ts

```diff
@@ -0,0 +1,144 @@
+import { ObjectLocale } from "src/commonTypes";
+
+interface qnadetail {
```

Comment:
interface 네이밍은 PascalCase로
```suggestion
interface QnADetail {
```

Reply:
resolved in 8572def54353f0dff07b6dc8da32b3dc665192b1